### ISSUE TITLE
Complete Texture Coverage

### ISSUE THREAD

**Issue by E-Roderick** [UQdeco2800/2021-studio-7#94]
## Description

This ticket focuses on the proposal of the creation of a large number of basic textures to better understand game scope, and ease work for Sprint 4. The textures will cover all aspects of the game, but will mainly focus on entities, walls, and floors (tiles). Work on textures should be divided amongst teams, to those that believe they can do more work. This should be a supplementary task, not a focus.

Any textures that are created should have undertaken a user testing process; however, as these textures are mostly for coverage, it is conceivable that the user testing will not be comprehensive.

## Textures

- Entities
  - _example..._
- Obstacles
  - _example..._
- Walls
  - Wall variation with windows
- Floors
  - Tile (material) and variation
  - Floor covered carpet and variation
  - Throw carpet textures
      - These textures would have a transparency and be placed on top of other floor tiles.
  - Default, exterior texture
      - For example, a darkened/faded dirt or grass texture.

## Documentation

This should include process ideation and user testing.

## Members

To be assigned after discussion.

**Comment by E-Roderick** [comment]
Please mention here any textures that should be added (or add them yourself), and if you or a team member would be available to work on them. I think decision on whether to work on this by 24/09 studio at the latest.

**Comment by tweeeb** [comment]
Hi! I added a lot of furniture entity textures as we will need to start furnishing the map rooms soon :)

**Comment by s4594749** [comment]
HAO LIANG TEAM5 GAME _LOOP:
i will help to do the 

**GAME Textures :**
Bin
Dining Table
Dining chairs
Lounge Furniture:
Lounge
Console (e.g. NES)
Bookcase

**and music sounds:** 
running sounds effect
walking sounds effect 
and replace the original background BGM

**Comment by tweeeb** [comment]
> 
> 
> HAO LIANG TEAM5 GAME _LOOP:
> I'm designer from team 5 and will help to do the
> 
> **GAME Textures:**
> 
> Dining Table
> Dining chairs
> Lounge Furniture:
> Lounge
> Console (e.g. NES)
> Bookcase
> 
> **and music sounds:**
> running sounds effect
> walking sounds effect
> and replace the original background BGM

 Thank you! Please let me know here when you have completed some of them :)

**Comment by s4594749** [comment]
> > HAO LIANG TEAM5 GAME _LOOP:
> > I'm designer from team 5 and will help to do the
> > **GAME Textures:**
> > Dining Table
> > Dining chairs
> > Lounge Furniture:
> > Lounge
> > Console (e.g. NES)
> > Bookcase
> > **and music sounds:**
> > running sounds effect
> > walking sounds effect
> > and replace the original background BGM
> 
> Thank you! Please let me know here when you have completed some of them :)



> > HAO LIANG TEAM5 GAME _LOOP:
> > I'm designer from team 5 and will help to do the
> > **GAME Textures:**
> > Dining Table
> > Dining chairs
> > Lounge Furniture:
> > Lounge
> > Console (e.g. NES)
> > Bookcase
> > **and music sounds:**
> > running sounds effect
> > walking sounds effect
> > and replace the original background BGM
> 
> Thank you! Please let me know here when you have completed some of them :)

Yes, we finished it, today we had a discussion about the size of the furniture, and now I have pushed my finished de into the main, you can see the images in the furniture file

**Comment by tweeeb** [comment]
Hi @s4594749 I noticed you have added the textures into the game (thank you so much!). Could you please upload them here as well (as a comment) just so there is a local collation of all the completed textures, it will also help when you link this issue to your sprint reflection. Thanks!

**Comment by s4594749** [comment]
![](https://raw.githubusercontent.com/UQdeco2800/2021-studio-7/main/source/core/assets/images/objects/furniture/diningtable.png)

**Comment by s4594749** [comment]
![](https://raw.githubusercontent.com/UQdeco2800/2021-studio-7/main/source/core/assets/images/objects/furniture/chair.png)

**Comment by E-Roderick** [comment]
I will try make some 'tile' textures for flooring.

**Comment by Noctisw** [comment]
Junxiang Wang
designer from team 2 
GAME Textures:
Bathroom: toilet, bath tub, sink and mirror, bin, shelf, tower hanger
Others: Nintendo console, stove, side table.
related wiki: https://github.com/UQdeco2800/2021-studio-7/wiki/Design-of-Furniture
